mkosi/gentoo.py

```diff
@@ -501,6 +502,8 @@ def invoke_emerge(
                                     f"actions={actions} outside stage3")
             emerge_main([*pkgs, *opts, *actions] + PREFIX_OPTS + self.emerge_default_opts)
         else:
+            if args.usr_only:
+                root_home(args, root).mkdir(mode=0o750, exist_ok=True)
```

Comment:
This should be handled by the change to prepare_tree(), why do this again here?

Reply:
See [here](https://github.com/systemd/mkosi/blob/fde6e2e264193600e105ab383775c6a51e7473f8/mkosi/gentoo.py#L328). The call to `invoke_emerge()` doesn't use the same root from `prepare_tree()`. Now we could run `root_home(args, stage3_tmp_extract).mkdir(mode=0o750)` before this but if ever there is another call to `invoke_emerge()` added later with a different value for root then this issue occurs again. I'm happy either way.

Reply:
Sure, let's keep it